src/components/Resources/Category/CategoriesGrid.js

```diff
@@ -29,6 +29,14 @@ export default function CategoriesGrid({ categories }) {
   return (
     <div>
       <Grid classes={gridStyles} container spacing={spacing}>
+        <Grid item>
```

Comment:
Would it be possible to separate the add category button into its own component, or reuse how the `categoryItems` list of components is created to append the add category button to the front of the list? We'd like the JSX after the return statement to show the structure of the component rather than the implementation.